src/TextField/TextFieldLabel.js

```diff
@@ -35,6 +55,11 @@ const propTypes = {
    * Override the inline-styles of the root element.
    */
   style: PropTypes.object,
+
```

Comment:
Let's hope they fix it https://github.com/yannickcr/eslint-plugin-react/pull/567.